beacon_node/src/config.rs

```diff
@@ -751,6 +751,10 @@ pub fn get_config<E: EthSpec>(
     client_config.chain.optimistic_finalized_sync =
         !cli_args.is_present("disable-optimistic-finalized-sync");
 
+    if cli_args.is_present("genesis-backfill") {
```

Comment:
Perhaps we should _also_ turn this on if the `--reconstruct-historic-states` flag is set?

Reply:
Yeah agreed.